### PR TITLE
feat: customize CTA

### PR DESCRIPTION
**Pull request by loriepisicchio**
## Issue

https://gravitee.atlassian.net/browse/APIM-2966

## Description

Customize CTA popup (title, button label, URL)

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-xwrkqjkpde.chromatic.com)

### CODE REVIEW

File: gravitee-apim-console-webui/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "@fontsource/libre-franklin": "4.4.5",
     "@gravitee/ui-analytics": "7.40.0",
     "@gravitee/ui-components": "3.45.2",
-    "@gravitee/ui-particles-angular": "7.48.0",
+    "@gravitee/ui-particles-angular": "7.48.0-optional-utm-params-in-cta-url-23f2c73",
```
Comment (loriepisicchio) [inline gravitee-apim-console-webui/package.json]: Requires this [PR](https://github.com/gravitee-io/gravitee-ui-particles/pull/328) to be merged first

File: gravitee-apim-console-webui/src/index.ts [hunk 2]
```diff
@@ -72,14 +72,14 @@ function fetchData() {
...
-      }
 
       const constants = _.assign(ConstantsJSON);
       constants.org.settings = consoleResponse.data;
-      if (uiCustomization?.data?.title) {
+
+      if (uiCustomization?.data) {
+        customizeUI(uiCustomization.data);
         constants.org.settings.management.title = uiCustomization.data.title;
+        constants.customization = uiCustomization.data;
```
Comment (jourdiw) [inline gravitee-apim-console-webui/src/index.ts:82]: If we add it to constants, then it won't be dynamic, right? I don't think that it's a big deal to have it in Constants. But if we add it there, then we can undo making the Menu Observables and just check the Constants.

File: gravitee-apim-console-webui/src/shared/components/gio-license/gio-license-data.ts [hunk 3]
```diff
@@ -46,70 +50,90 @@ export enum UTMTags {
...
+    },
+    [ApimFeature.APIM_SCHEMA_REGISTRY_PROVIDER]: {
+      title,
+      image: 'assets/gio-ee-unlock-dialog/confluent-schema-registry.svg',
+      description: `Confluent Schema Registry is part of ${ee}. Integration with a Schema Registry enables your APIs to validate schemas used in API calls, and serialize and deserialize data.`,
+      trialButtonLabel,
+      hideDays,
+    },
+    [ApimFeature.APIM_EN_MESSAGE_REACTOR]: {
+      title: 'Request an upgrade', // FIXME: should we keep this custom title here?
```
Comment (jourdiw) [inline gravitee-apim-console-webui/src/shared/components/gio-license/gio-license-data.ts:125]: I feel like this should be customizable. But maybe for M2.

**Review by jourdiw — APPROVED** [review]

File: gravitee-apim-console-webui/src/index.ts (hunk 2 above)
Comment (loriepisicchio) [reply gravitee-apim-console-webui/src/index.ts:82]: You're right. I have simplified the code to use Constants

**Review by Okhelifi — APPROVED** [review]

**Pull request merged**